script.js

```diff
@@ -0,0 +1,140 @@
+    profession: 'Ebolom is the Executive Director of the Wikimedia Foundation, the nonprofit organization that operates Wikimedia. Wikimedia is a freely available news article on health matters and all things health, written in 290 languages and is used by nearly half a billion people around the world every month.',
+    image: 'images/profile.jpg',
+  },
+  {
+    id: 4,
+    name: 'John Collins',
+    qualification: 'Internationally reknown and successful Physician and physical wellness activist',
+    profession: 'Collins studies Physical wellness, and published his seminal book The Fit Body & Mind in 2011.',
+    image: 'images/profile.jpg',
+  },
+  {
+    id: 5,
+    name: 'Catherine Nancy',
+    qualification: 'Berkman Professor of Medicine and Biological Health Studies at Strathmore University',
+    profession: 'Nancy studies Biology and Advanced Genetics, and published the most sought after fitness educational article, The Power of Biology in 2021',
+    image: 'images/profile.jpg',
+  },
+  {
+    id: 6,
+    name: 'Akachukwu Smart',
+    qualification: 'Berkman Professor of Biological Studies at Harvard Medical School',
+    profession: 'Smart is an established and successful proactive body builder who has his career set on body building and is thus one of the best candidates to discuss matters physical health',
+    image: 'images/profile.jpg',
+  },
+];
+presenters.forEach((presenter) => {
+  const list = document.createElement('li');
+  list.classList.add(`presenter${presenter.id}`);
+  list.innerHTML = `
+        <div class="speaker-image">
+            <div class="speaker-bg"></div>
+            <img src="${presenter.image}" alt="Presenter Image">
+        </div>
+        <div class="speaker-info">
+            <h3>${presenter.name}</h3>
+            <h5>${presenter.qualification}</h5>
+            <div></div>
+            <p>${presenter.profession}</p>
+        </div>
+    `;
+  presentation.appendChild(list);
+});
+
+// See Program
+
+const seeLess = document.querySelector('#see-less');
+const seeMore = document.querySelector('#see-more');
+const partners = document.querySelector('#partners');
+const footer = document.querySelector('#footer');
+const seeDesktop = document.querySelector('#see-program');
+// const speakersList = document.querySelectorAll('#speakers-list');
+
+function toggleHideMobile() {
+  speakers.classList.add('hidden');
+  partners.classList.add('hidden');
+  footer.classList.add('hidden');
+  seeLess.classList.add('hidden');
+  seeMore.classList.remove('hidden');
+  // speakersList.forEach((list) => {
+  //     list.classList.add('hidden');
+  // });
+}
+
+function toggleSeeMobile() {
+  speakers.classList.remove('hidden');
+  partners.classList.remove('hidden');
+  footer.classList.remove('hidden');
+  seeLess.classList.remove('hidden');
+  seeMore.classList.add('hidden');
+  // speakersList.forEach((list) => {
+  //     list.classList.remove('hidden');
+  // });
```

Comment:
- Please, let's remove the commented code to make our codebase clean and readable.